core/modules/layout/layout.module

```diff
@@ -568,15 +568,16 @@ function layout_route_handler($router_item) {
   // Safety check that we don't show an admin layout to a user who cannot view
   // the admin theme. This may happen because the cached layout list does not
   // include a permissions check on the admin theme.
-  if ($selected_layout->name === 'admin_default' && !user_access('view the administration theme')) {
+  $acess_admin_theme = user_access('view the administration theme');
```

Comment:
`acess_admin_theme` should be `access_admin_theme` everywhere. Also correct "specail".

Reply:
```suggestion
  $access_admin_theme = user_access('view the administration theme');
```